javascript-carousel/main.js

```diff
@@ -0,0 +1,94 @@
+const $image = document.querySelector('.carousel-img');
+const allCircles = document.querySelectorAll('.fa-circle');
+const $circle1 = document.querySelector('#circle-1');
+const $circle2 = document.querySelector('#circle-2');
+const $circle3 = document.querySelector('#circle-3');
+const $circle4 = document.querySelector('#circle-4');
+const $circle5 = document.querySelector('#circle-5');
+const $leftChevron = document.querySelector('.fa-chevron-left');
+const $rightChevron = document.querySelector('.fa-chevron-right');
+const $dotRow = document.querySelector('.dots');
+let pokemonInterval = setInterval(loopForward, 1500);
+
+function loopForward() {
+  if ($image.getAttribute('src') === 'images/001.png') {
+    $image.setAttribute('src', 'images/004.png');
+    $circle1.setAttribute('class', 'fa-regular fa-circle');
+    $circle2.setAttribute('class', 'fa-solid fa-circle');
+  } else if ($image.getAttribute('src') === 'images/004.png') {
+    $image.setAttribute('src', 'images/007.png');
+    $circle2.setAttribute('class', 'fa-regular fa-circle');
+    $circle3.setAttribute('class', 'fa-solid fa-circle');
+  } else if ($image.getAttribute('src') === 'images/007.png') {
+    $image.setAttribute('src', 'images/025.png');
+    $circle3.setAttribute('class', 'fa-regular fa-circle');
+    $circle4.setAttribute('class', 'fa-solid fa-circle');
+  } else if ($image.getAttribute('src') === 'images/025.png') {
+    $image.setAttribute('src', 'images/039.png');
+    $circle4.setAttribute('class', 'fa-regular fa-circle');
+    $circle5.setAttribute('class', 'fa-solid fa-circle');
+  } else {
+    $image.setAttribute('src', 'images/001.png');
+    $circle5.setAttribute('class', 'fa-regular fa-circle');
+    $circle1.setAttribute('class', 'fa-solid fa-circle');
+  }
+}
```

Comment:
This works, but it makes this really hard to be dynamic, because you're hard coding what image paths you want to check for. What if you decided you wanted to add 10 more images? Then you would need to back and add `if` statemnts for all of those.

If you did something with a `count` variable instead, this would have been much simpler. Then you could utilize your counter to know essentially what index of the `img` or `i` you were on, and show the correct image or circle